src/Stepper/Stepper.js

```diff
@@ -23,9 +23,9 @@ class Stepper extends Component {
      */
     activeStep: PropTypes.number,
     /**
-     * Should be two or more `<Step />` components
+     * Should be two or more `<Step />` components, but also accepts one.
```

Comment:
How do you mean exactly? I tried to preserve the comment as "this component should probably have at least two children, but if you're absolutely sure you can use one child"